casepro/msgs/forms.py

```diff
@@ -121,6 +121,9 @@ def clean_labels(self):
         if parent is None and labels is None:
             raise forms.ValidationError(_("Labels are required if no Parent is selected"))
 
+        if labels is None:
```

Comment:
This is kinda odd because `labels` is assigned `None` just above. I think it would be cleaner to use `derive_fields` on the view to remove labels as a field altogether when editing a translation.